requirements.txt

```diff
@@ -5,6 +5,7 @@ diff_match_patch_python ~=1.0.2
 docopt ~=0.6.2
 git+https://github.com/anastasia/htmldiffer@develop
 git+https://github.com/danielballan/htmltreediff@customize
+html5-parser ~=0.4.7
```

Comment:
Per @danielballan’s [comment](https://github.com/edgi-govdata-archiving/web-monitoring-processing/pull/410#issuecomment-495080497), I think this still needs `--no-binary lxml` on the end.

Reply:
@danielballan that’s how this should be set up, right?

Reply:
@Mr0grog when I do that, I get an error. Check out:
https://travis-ci.org/edgi-govdata-archiving/web-monitoring-processing/builds/541947661?utm_source=github_status&utm_medium=notification

Reply:
Seems like we need to do some better parsing in `setup.py` to support this: https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/19d9de2bc558a10851601487bb34c7478cde4747/setup.py#L22-L25

Reply:
@Mr0grog I'm sorry but I'm a bit confused.
What do you suggest to do here? Remove `--no-binary lxml`?

Reply:
Well, the [format of lines in a pip file](https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format) is:

```
<requirement specifier> [; markers] [[--option]...]
```

Whereas `install_requires` in `setup.py` can only have `<requirement specifier>`. So it seems like the right move is to remove anything after the specifier.

It’s not perfectly accurate, but the easiest thing might be to just look for ` ;` or ` --` on the line and, if present, remove that string and everything after it. That would probably do the job well enough for our purposes, at least.

Reply:
OK, its done! Thank you for the explanation @Mr0grog . Could you please merge now?

Reply:
Looks like you are still some linting errors, but will take a look and see about merging when I am off work tonight or tomorrow.

Reply:
Linting error fixed!